### ISSUE TITLE
nccopy ignores chunkspec on linux (but not under windows)

### ISSUE THREAD

**Issue by wstkisters** [Unidata/netcdf-c#1817]
Hello!

**Version**
netCDF library version 4.7.4 of Apr 10 2020 23:29:06 under Linux 5.7.14-1-MANJARO 1 SMP PREEMPT Fri Aug 7 10:12:32 UTC 2020.

**How to reproduce:**
The command 
`nccopy -d1 -c time/50,lat/50,lon/50 in.nc out.nc `
ignores the chunkspec (any chunksize, any combination), according to the output of 
`ncdump -sh out.nc`

However, running 
```
nccopy -3 in.nc tmp.nc 
nccopy -d1 -c time/50,lat/50,lon/50 tmp.nc out.nc 
```
produces a chunked netcdf file. 

**Notes**:
The same command 
`nccopy -d1 -c time/50,lat/50,lon/50 in.nc out.nc `
under Windows 10, netCDF library version 4.7.3 of Feb  3 2020 16:15:18 $ produces the desired result

**Comment by DennisHeimbigner** [comment]
Do you have a cdl file containing the metadata for in.nc (I don't
need the data, only the metadata). Use ncdump -h in.nc to get it.

**Comment by wstkisters** [comment]
Had to rename the file ending so that github accepts it:

[in.log](https://github.com/Unidata/netcdf-c/files/5122203/in.log)

Thanks

**Comment by DennisHeimbigner** [comment]
I forgot to ask; is the original in.nc a netcdf-4 file?

**Comment by wstkisters** [comment]
Yes, netcdf4-classic. I also tried with netcdf4, doesn't work either.

[in.log](https://github.com/Unidata/netcdf-c/files/5128300/in.log)
